packages/x-archetype-utils/src/home/home-template.html

```diff
@@ -1706,6 +1702,7 @@ <h1 class="x-archetype-section1__title x-margin--top-05 x-margin--bottom-07">
 
     --x-archetype-color-base-neutral-10: #283034;
     --x-archetype-color-lead: #243d48;
+    --x-archetype-button-color: #5e7782;
```

Comment:
isn't this color different than the one we are using in the archetype? Also, shouldn't the icon have `border-radius` instead of being square? Ii is only visible when you zoom in, though.

`home-template`
![image](https://user-images.githubusercontent.com/6247440/211856861-8e649e4f-1ecf-411a-b370-3b0b10b56d88.png)

`archetype`
![image](https://user-images.githubusercontent.com/6247440/211856938-9e07f3d2-79be-4c9f-9779-7c4db94ac254.png)

Reply:
@LuisMartinez15 I'm checking [Figma](https://www.figma.com/file/mYfJZHn668mGEGf61Sji6m/X-Archetype?node-id=5541%3A83202&t=vomH4eTjmAU5fgOo-0) and the color and icon size are the ones we have in the designs. 
Agree with border-radius, set to `border-radius: 0px 4px 4px 0px;` in designs.

Reply:
In figma, the home page has these styles:

![image](https://user-images.githubusercontent.com/6597815/212007833-257a1d27-489b-4a77-94f9-216fa25aec8d.png)

in the home-template:

![image](https://user-images.githubusercontent.com/6597815/212007962-bfdc1624-7ffa-4db7-9b43-de03fcf76461.png)

![image](https://user-images.githubusercontent.com/6597815/212007694-d7c80596-15f5-4766-8593-5fe6fed313df.png)

The color and size are differents

Reply:
Color is changed because the color in the designs doesn't align with the color of the buttons in the archetype. If you use the template with the color in the designs in the achetype the button color "changes" and it's weird.

I'll check the border radius.